src/store/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SET_DATA: string = 'SET_DATA';
+export type SET_DATA = typeof SET_DATA;
```

Comment:
what does it mean? You are exporting two things with the same name from same file. What's the intention to do so?